src/helperFunctions/toast.ts

```diff
@@ -0,0 +1,12 @@
+import { toast } from 'react-toastify';
+
+type ToastType = 'info' | 'success' | 'warning' | 'error' ;
+
+const Toast = (type: ToastType, message: string) => toast[type](message, {
```

Comment:
Any reason for the capital first letter? 🤔 
If not please make it lowercase, we follow the `camelCase` variable name in this codebase.

Reply:
naming it as **toast** gives an error as _import declaration conflicts with local declaration of 'toast'._

Reply:
Addressed the issue in the comment on line number 1